src/routes/api/v1/follow.js

```diff
@@ -9,6 +11,19 @@ router
   .all(asyncHandler(checkAuth))
   .all(asyncHandler(checkUsername))
   .post(asyncHandler(FollowController.follow))
-  .delete(asyncHandler(FollowController.unFollow));
+  .delete(asyncHandler(FollowController.unFollow))
+
+router
+  .route('/:username/all')
```

Comment:
The endpoint should instead be `/profiles/:username/followers` because followers are first related to the profile then the user